app/policies/project_policy.rb

```diff
@@ -14,7 +14,7 @@ def create?
   end
 
   def update?
-    user.id == project.user_id
+    project.user_id.in?(project.users.ids)
```

Comment:
Don't use `in?` (https://rails.rubystyle.guide/#active_support_extensions)
```suggestion
    project.user_ids.include?(project.user_id)
```